components/Breadcrumbs.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { Dispatch, SetStateAction, useEffect, useState } from 'react'
+import { useRouter } from 'next/router'
+import Link from 'next/link'
+import styles from '../scss/breadcrumbs.module.scss'
+import { Dropdown } from 'react-bootstrap'
+import { GetAppQuery } from '../graphql'
+import { ChevronRightIcon } from '@primer/octicons-react'
+import { lowerCase } from 'lodash'
+
+type Props = {
+  omitHomeRoute?: boolean
+  homeTitle?: string
+  lessonTitle: string
+  setLessonTitle: Dispatch<SetStateAction<string>>
+  lessons: GetAppQuery['lessons'] | undefined
+}
+
+type Breadcrumb = {
```

Comment:
Is the rest of codebase using `type` or `interface`. Whichever ever it is, let's make sure we pick one and be consistent.

Reply:
Are you suggesting to stick to using just `type` or `interface` across the whole codebase in every scenario? Or are you suggesting to stick to using `type` or `interface` only when declaring type definitions for objects?